changelog-2023.08.18-8.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index 4a95df972b..5384ba774f 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -113,6 +113,32 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2023.08.18-8" date="2023-08-18">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>New OpenStreetMap data as of August 14, 2023</li>
```

Comment:
• New OpenStreetMap maps data as of August 14
• Updated more than a year old Wikipedia articles
• Added missing isolines for some regions of Argentina, Chile, Kazakhstan, Tibet, USA, Russia (map sizes increased)
• More dense isolines in many areas
• Removed too small isolines at overview zoom levels
• Reduced many other map sizes
• Import comments from GPX cmt tag
• Treat GPX track segments as part of the singular track they belong to
• Fixed a bug when editing social contacts
• Dog Park can be added in the Editor
• Updated Persian voice directions

Routing
• Updated road surface factors for car routing profile
• Better pedestrian routing in Turkey and Georgia
• Fixed cross-region routing when "avoid …" options are enabled

Search
• Fixed search for addresses in Czech and Slovakia
• Improved street matching
• Remove Tourist Information, Place of Worship, Park, Information Board, and Tourist Map from the Sightseeings category
• Do not filter duplicated search results except for very close bus stops
• Better ranking when searching by category
• Fixed search by a house name
• Faster search on the map screen

Styles
• Added ferry routes to the World map
• Removed square background from parking icons
• Added separate icons for paid and private parking
• Made the bicycle parking icon differ more from the motorcycle one
• Updated parking entrance icon
• Street-side and lane parkings use a smaller icon and appear later when zooming in
• Added motorcycle parking and parking entrance icons to the navigation mode
• Fixed motorway=trunk links visibility in navigation mode
• Improved motorway=trunk colors
• Added Tehran metro icon
• Revamped priorities system, especially for POIs

Reply:
Thanks added.